tests/test.py

```diff
@@ -1638,14 +1671,13 @@ def run_aws(*, image: str, user: str) -> None:
     LOG.info(f"{image} passed {num_tests_ran} integration tests as {user}")
 
 
-def run_security(*, tag: str) -> None:
+def run_security(*, tool: str, environment: str, tag: str) -> None:
     """Run the security tests"""
-    sbom_file = Path(f"sbom.{tag}.json")
+    sbom_file: Path = Path(f"sbom.{tag}.json")
 
     if not sbom_file:
-        LOG.error(
-            f"{sbom_file} was not found; security scans require an SBOM. Please run `task sbom`"
-        )
+        LOG.debug("Generating an SBOM...")
+        utils.sbom(tool=tool, environment=environment)
```

Comment:
This should fix issues like [this](https://github.com/SeisoLLC/easy_infra/actions/runs/6017565636/job/16324048731#step:9:148)

```bash
	* failed to catalog: unable to open file sbom.2023.08.15-terraform.json: open sbom.2023.08.15-terraform.json: no such file or directory
```

From cab3e5a31b17e4649bff66b9b27722ee96a09886